src/java/org/apache/cassandra/metrics/KeyspaceMetrics.java

```diff
@@ -82,17 +82,17 @@ public class KeyspaceMetrics
     /** Tombstones scanned in queries on this Keyspace */
     public final Histogram tombstoneScannedHistogram;
     /** Time spent flushing memtables */
-    public final Histogram flushTime;
-    public final Histogram storageAttachedIndexBuildTime;
+    public final Counter flushTime;
+    public final Counter storageAttachedIndexBuildTime;
 
     /** Time spent writing SAI */
-    public final Histogram storageAttachedIndexWritingTimeForIndexBuild;
-    public final Histogram storageAttachedIndexWritingTimeForCompaction;
-    public final Histogram storageAttachedIndexWritingTimeForFlush;
-    public final Histogram storageAttachedIndexWritingTimeForOther;
```

Comment:
Aren't we losing the information about the distribution of time of each flush / compaction / index write?
I feel like the individual operation size is also important, not only the total size.
Have you considered leaving them and just adding the counters (if we can't get the totals from histograms)?

Reply:
These come from my previous tentative: these histograms are pretty ueseless in practice, as I have seen in Grafana.
It is hard to correlate the various "updates" to the Histograms

Reply:
the original goal to have a "percentage" or the time spent building SAI vs the main operation (flush/compaction...) cannot be achieved with the histograms